README.md

```diff
@@ -20,13 +20,22 @@
 
 To get a local copy just download the repository ZIP, UNZIP then open the Index.html file.
```

Comment:
**Change Required**
- [ ] The `live version` link is still pointing to an outdated link. Let us remember to update our README file for every change since it acts as a documentation for others.